R/map.R

```diff
@@ -228,18 +225,11 @@ mmap_ <- function(.x, .f, .progress, .type, error_call, ...) {
   }
   if (...length()) {
     cli::cli_abort(
-      "Don't use `...` with `.parallel = TRUE`.",
+      "Don't use `...` with parallelized functions.",
```

Comment:
```suggestion
      "Can't use `...` with parallelized functions.",
```

https://style.tidyverse.org/errors.html#problem-statement